web/src/components/organisms/Project/Content/ContentDetails/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+
+import ContentDetailsMolecule from "@reearth-cms/components/molecules/Content/ContentDetailsMolecule";
+import ModelsMenu from "@reearth-cms/components/organisms/Project/ModelsMenu";
+import { useT } from "@reearth-cms/i18n";
+
+import useHooks from "./hooks";
+
+const ContentDetails: React.FC = () => {
+  const { projectId, workspaceId, itemId, modelId } = useParams();
+  const { currentModel, handleItemCreate, handleItemUpdate, initialFormValues, defaultFormValues } =
+    useHooks({ itemId });
+  const t = useT();
+
+  const navigate = useNavigate();
+
+  const selectModel = useCallback(
+    (modelId: string) => {
+      navigate(`/workspaces/${workspaceId}/${projectId}/content/${modelId}`);
+    },
+    [navigate, workspaceId, projectId],
+  );
```

Comment:
This navigates somewhere. It doesn't select a model. Similar to the callback below, perhaps handleNavigateToModel or something is clearer

Reply:
Renamed to `handleNavigateToModel`.